geomesa-tools/README.md

```diff
@@ -229,8 +234,259 @@ To remove a feature type and it's associated data from a catalog table, use the
 ####Example:
     geomesa removeschema -u username -p password -i instance -z zoo1,zoo2,zoo3 -c test_catalog -f testfeature1
     geomesa removeschema -u username -p password -i instance -z zoo1,zoo2,zoo3 -c test_catalog --pattern 'testfeatures\d+'
-    
-    
+
+
+### stats-analyze
```

Comment:
There's a shorter summary of the command line arguments in ``docs/user/commandline_tools.rst``, but as @elahrvivaz says, it's already updated in this PR.